kong/plugins/prometheus/prometheus.lua

```diff
@@ -356,6 +372,11 @@ function Prometheus.init(dict_name, prefix)
   return self
 end
 
+-- enable the use the lua-resty-counter for Counter and Histogram
+function Prometheus:use_resty_counter(counter)
```

Comment:
Better rename to`set_resty_counter` here?

Reply:
Updated!